cdc/owner_test.go

```diff
@@ -74,6 +76,25 @@ func (s *ownerSuite) TearDownTest(c *check.C) {
 	}
 }
 
+type mockPDClient struct {
+	pd.Client
+}
+
+func (m *mockPDClient) UpdateServiceGCSafePoint(ctx context.Context, serviceID string, ttl int64, safePoint uint64) (uint64, error) {
+	return 0, errors.New("mock error")
+}
+
+func (s *ownerSuite) TestOwnerFlushChangeFeedInfos(c *check.C) {
+	mockOwner := Owner{
+		pdClient:              &mockPDClient{},
+		gcSafepointLastUpdate: time.Now(),
+	}
+
+	// Owner should ignore UpdateServiceGCSafePoint error.
+	err := mockOwner.flushChangeFeedInfos(s.ctx)
+	c.Assert(err, check.IsNil)
```

Comment:
Recommend adding the check that `mockOwner.gcSafepointLastUpdate` is not updated